Makefile

```diff
@@ -53,4 +53,8 @@ clean: ## Cleans all unversioned files in project
 
 .PHONY: debug-github-workflow
 debug-github-workflow: ## Runs the github worflow locally act is needed on the system
-	@act -v -s DOCKER_HUB_USER=docke_hub_username -s DOCKER_HUB_PASSWORD=docke_hub_password
\ No newline at end of file
+	@act -v \
```

Comment:
I am an not mistaken, now the requirements to run all recipies the Makefile recipes of this repo are:
- make
- awk
- docker
- docker-compose
- python3
- act <-- NEW

a) Can i reduce these dependencies e.g. only to docker/docker-compose?
b) If not, can we add scripts to install each of them in place. Ideally, I would clone the repo, install my dev-tools and start to work

Reply:
**act** is only used for local testing of GitHub actions. I would add a comment with a link on how to install it.